dockstore-event-consumer/src/license/THIRD-PARTY.properties

```diff
@@ -0,0 +1 @@
+io.dockstore--zenodo-client-generated--0.1=The Apache Software License, Version 2.0
```

Comment:
1. Is this our Dockstore Zenodo client (swagger-java-zenodo-client)? If so, isn't the fix to ensure that it has a license? I don't know why it's not inheriting the license from the root pom.xml.
2. If you need this, and regarding dockstore-integration-testing/src/license/THIRD-PARTY.properties, is it possible to have the values in the root THIRD-PARTY properties? I feel like it would be easier to manage these in one place.

Reply:
The license is picked up from the parent pom.xml for:
     (Apache Software License, Version 2.0) swagger-java-zenodo-client (io.dockstore:swagger-java-zenodo-client:1.9.0-alpha.6-SNAPSHOT - https://github.com/dockstore/dockstore/swagger-java-zenodo-client)

But not for
     (The Apache Software License, Version 2.0) zenodo-client-generated (io.dockstore:zenodo-client-generated:0.1 - no url defined)
which has to be added manually

Reply:
I cannot find a way to use a THIRD-PARTY.properties file from the root directory. The plugin seems to want to use a path in each module dir. However there might be a way to specify a file for missing license information using a URL.